src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,40 @@
+.catalog {
+  grid-area: main;
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  justify-content: center;
+  grid-gap: $gap-size;
+
+  &__card {
+    display: grid;
+    justify-items: center;
+    align-items: center;
+    height: $card-height;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+}
+
+@media (max-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(1, $card-width);
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
```

Comment:
you can move these media queries inside the .catalog block, then you won't need to write a selector every time